content/en/docs/tutorials/using_breakpoints_to_debug_tests.md

```diff
@@ -214,54 +226,59 @@ start(_Config) ->
     ?assertEqual(1, proplists:get_value(workers, Props)),
     ok.
 ```
-In this version, we not only set the breakpoint after checking for the children, we also only trigger it when a result is unexpected. This can be useful when you have some tests that do not always fail, only sometimes. Since breakpoints are normal function call, you can even leave them in place with instructions to start debugging at this point.
+
+In this version, we not only set the breakpoint after checking for the children, we also only trigger it when a result is unexpected. This can be useful when you have some tests that do not always fail, only sometimes. Since breakpoints are normal function calls, you can even leave them in place with instructions to start debugging at this point.
 
 Let's run the test to see it stall on our breakpoint:
 
-	$ rebar3 shell --start-clean
-	...
-	1> r3:async_do(ct).
-	===> This feature is experimental and may be modified or removed at any time.
-	ok
-	2> Verifying dependencies...
-	Compiling break_check
-	Running Common Test suites...
-	%%% start_SUITE:
-	=== BREAK ===
+```erlang
+$ rebar3 shell --start-clean
+...
+1> r3:async_do(ct).
+===> This feature is experimental and may be modified or removed at any time.
+ok
+2> Verifying dependencies...
+Compiling break_check
+Running Common Test suites...
+%%% start_SUITE:
+=== BREAK ===
+```
 
 Now we can look through the system. The application is started, the supervisor is running, and we can manipulate things, such as starting a fake child to see if that would fix the test:
 
-	%% Poke at the supervisor's state and see it has no children
-	2> sys:get_state(break_check_sup).
-	{state,{local,break_check_sup},
-	       one_for_all,
-	       {[],#{}},
-	       undefined,0,1,[],0,break_check_sup,[]}
-	
-	%% Add a child by hand
-	3> supervisor:start_child(
-	3>   break_check_sup,
-	3>   #{id => some_child,
-	3>     start => {gen_event, start_link, []},
-	3>     type => worker}
-	3> ).
-	{ok,<0.232.0>}
-	
-	%% Check that it is actually tracked
-	4> sys:get_state(break_check_sup).
-	{state,{local,break_check_sup},
-	       one_for_all,
-	       {[some_child],
-	        #{some_child =>
-	              {child,<0.232.0>,some_child,
-	                     {gen_event,start_link,[]},
-	                     permanent,5000,worker,
-	                     [gen_event]}}},
-	       undefined,0,1,[],0,break_check_sup,[]}
-	
-	%% and keep going to see the test pass
-	5> r3:resume().
-	ok
-	All 1 tests passed.
-	 
-It would be a good idea to remove all of your breakpoints after you're done with them, but it's interesting to know you can use features such as Common Test's [repeated executions with groups ](http://erlang.org/doc/apps/common_test/write_test_chapter.html#test-case-groups) or randomized runs in order to detect weird conditions and automatically trigger a break point when required.
+```shell
```

Comment:
is shell or erlang a better language for these for syntax highlights?

Reply:
I tried to choose based on the percentage of what I found most:
* `shell` should be used for e.g. bash stuff,
* `erlang` should be used for actual Erlang code.

I'll re-review all of these, too. I got into the `markdownlint` craze and some might be off.

Reply:
All of these were review in a commit mentioned above.